src/main/java/de/nadirhelix/guestbook/GuestbookApplication.java

```diff
@@ -1,13 +1,28 @@
 package de.nadirhelix.guestbook;
 
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 
 @SpringBootApplication
 public class GuestbookApplication {
 
+	private static final String DEV_MODE_KEY = "-DdevMode=";
```

Comment:
Spring Boot kann Args direkt mappen: https://memorynotfound.com/spring-boot-passing-command-line-arguments-example/.
Du würdest also in einer Bean, wo du den Wert brauchst zum Beispiel ein "@Value("${is.dev}") private boolean isDev" machen und dann das ganze mit "--is.dev=true" starten.

Wäre das eventuell eine Alternative?

Mit diesem Vorgehen kann man generell alle "application properties" sich in Beans mappen und beim Start der Anwendung überschreiben/angeben.

So funktioniert auch das Setzen des Ports beim Starten: --server.port=8080.